libs/platform/i18n/src/lib/locale/adapter/default-locale.adapter.ts

```diff
@@ -0,0 +1,39 @@
+import { inject } from '@spryker-oryx/di';
+import { Observable, of } from 'rxjs';
+import { Locale } from '../../models';
+import { LocaleAdapter } from './locale.adapter';
+
+export class DefaultLocaleAdapter implements LocaleAdapter {
+  constructor(
+    protected readonly config = inject(DefaultLocaleAdapterConfig, null)
+  ) {}
+
+  getAll(): Observable<Locale[]> {
+    return of(this.config?.locales ?? []);
+  }
+
+  getDefault(): Observable<string> {
+    return of(
+      this.config?.defaultLocale ??
+        this.config?.locales?.[0].code ??
+        this.getFallbackLang()
+    );
+  }
+
+  protected getFallbackLang(): string {
+    return navigator.language;
```

Comment:
won't work on SSR?

Reply:
No it won't, but if you don't want it to get called - you should manually provide "default" locale instead.